### PR TITLE
Add regulatory compliance cmdlets and tests

### PR DESCRIPTION
**Pull request by amitmag-ms1**
## Description

Add new cmdlets to support Regulatory Compliance API

## Checklist

- [X ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X ] The title of the PR is clear and informative
- [X ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X ] If applicable, the changes made in the PR have proper test coverage
- [X ] For public API changes to cmdlets:
    - [ X] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [X ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/Accounts/Accounts/Accounts.csproj [hunk 1]
```diff
@@ -29,6 +29,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```
Comment (msJinLei) [inline src/Accounts/Accounts/Accounts.csproj:32]: Why add new package reference in Account module?

File: src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj [hunk 2]
```diff
@@ -11,6 +11,10 @@
     <RootNamespace>$(LegacyAssemblyPrefix)Common.Authentication.ResourceManager</RootNamespace>
   </PropertyGroup>
 
+    <ItemGroup>
+      <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```
Comment (msJinLei) [inline src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj:15]: Why add new package reference here?

File: src/Accounts/Authentication/Authentication.csproj [hunk 3]
```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.SecurityCenter" Version="1.1.1" />
```
Comment (msJinLei) [inline src/Accounts/Authentication/Authentication.csproj:15]: Why add new package reference here?

File: src/Accounts/Accounts/Accounts.csproj (hunk 1 above)
Comment (amitmag-ms) [reply src/Accounts/Accounts/Accounts.csproj:32]: This was automatically added after updating the SecurityCenter nuget

File: src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj (hunk 2 above)
Comment (amitmag-ms) [reply src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj:15]: This was automatically added after updating the SecurityCenter nuget

File: src/Accounts/Authentication/Authentication.csproj (hunk 3 above)
Comment (amitmag-ms) [reply src/Accounts/Authentication/Authentication.csproj:15]: This was automatically added after updating the SecurityCenter nuget

**Comment by amitmag-ms** [comment]
Link for the design review: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/500

File: src/Accounts/Accounts/Accounts.csproj (hunk 1 above)
Comment (amitmag-ms1) [reply src/Accounts/Accounts/Accounts.csproj]: Removed

Comment (amitmag-ms1) [reply src/Accounts/Accounts/Accounts.csproj]: This was automatically added after updating the SecurityCenter nuget. Removed.

File: src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj (hunk 2 above)
Comment (amitmag-ms1) [reply src/Accounts/Authentication.ResourceManager/Authentication.ResourceManager.csproj]: This was automatically added after updating the SecurityCenter nuget. Removed.

File: src/Accounts/Authentication/Authentication.csproj (hunk 3 above)
Comment (amitmag-ms1) [reply src/Accounts/Authentication/Authentication.csproj]: This was automatically added after updating the SecurityCenter nuget. Removed.

File: src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md:39]: Example cannot be empty.

File: src/Security/Security/help/Get-AzRegulatoryComplianceControl.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplianceControl.md:40]: Example cannot be empty.

File: src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+## DESCRIPTION
+{{ Fill in the Description }}
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md:34]: Example cannot be empty.

File: src/Security/Security/Cmdlets/RegulatoryCompliance/GetRegulatoryComplianceControl.cs [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+namespace Microsoft.Azure.Commands.SecurityCenter.Cmdlets.RegulatoryCompliance
+{
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "RegulatoryComplianceControl", DefaultParameterSetName = ParameterSetNames.SubscriptionLevelResource), OutputType(typeof(PSSecurityRegulatoryComplianceControl))]
+    public class GetRegulatoryComplianceControl : SecurityCenterCmdletBase
+    {
+        [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = false, HelpMessage = ParameterHelpMessages.ResourceName)]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.SubscriptionLevelResource, Mandatory = true, HelpMessage = ParameterHelpMessages.StandardName)]
```
[the remark this answers is not in the document]
Comment (amitmag-ms1) [reply src/Security/Security/Cmdlets/RegulatoryCompliance/GetRegulatoryComplianceControl.cs:30]: Removed

File: src/Security/Security/Cmdlets/Alerts/SetAlerts.cs [hunk 8]
```diff
@@ -80,8 +81,8 @@ public override void ExecuteCmdlet()
                         case "dismissed":
                             actionType = "Dismiss";
                             break;
-                        case "active":
```
[the remark this answers is not in the document]
Comment (amitmag-ms1) [reply src/Security/Security/Cmdlets/Alerts/SetAlerts.cs]: It suppose to fix an issue in another cmdlet, but it was already fixed in different PR so I can remove it. Removed.

File: src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md (hunk 4 above)
Comment (amitmag-ms1) [reply src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md:39]: Added 3 different examples.

File: src/Security/Security/help/Get-AzRegulatoryComplianceControl.md (hunk 5 above)
Comment (amitmag-ms1) [reply src/Security/Security/help/Get-AzRegulatoryComplianceControl.md:40]: Added 3 different examples.

File: src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md (hunk 6 above)
Comment (amitmag-ms1) [reply src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md:34]: Added 3 different examples.

File: src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md [hunk 9]
```diff
@@ -0,0 +1,272 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplainceAssessment.md:4]: Please refer to the other cmdlet help file to fill the "online version"

File: src/Security/Security/help/Get-AzRegulatoryComplianceControl.md [hunk 10]
```diff
@@ -0,0 +1,200 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplianceControl.md:4]: online version cannot be empty

File: src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Security.dll-Help.xml
+Module Name: Az.Security
+online version:
```
Comment (msJinLei) [inline src/Security/Security/help/Get-AzRegulatoryComplianceStandard.md:4]: online version cannot be empty

**Pull request merged**